README.md

```diff
@@ -84,8 +84,15 @@ Please check [here](os.md) for more information about which operating system to
 
 ## Postgres
 Postgres performance can be heavily impacted by not using [LSE](https://github.com/aws/aws-graviton-getting-started/blob/main/c-c%2B%2B.md#large-system-extensions-lse).
-Today, postgres binaries from distributions (e.g. Ubuntu) are not built with `-moutline-atomics` or `-march=armv8.2-a` which would enable LSE. If you're planning to use
-postgres in production, please rebuild it with flags to enable LSE. Note: Amazon RDS for PostgreSQL isn't impacted by this. 
+Today, postgres binaries from distributions (e.g. Ubuntu) are not built with `-moutline-atomics` or `-march=armv8.2-a` which would enable LSE.  Note: Amazon RDS for PostgreSQL isn't impacted by this. 
+
+Update: PostgreSQL started to distribute Ubuntu 20.04 packages optimized with `-moutline-atomics`.
```

Comment:
no one knows why this is an updated and whet it was an updated from. Just say, Starting on XX,yy ...